dhcpv6/nclient6/client.go

```diff
@@ -470,7 +471,7 @@ func (c *Client) SendAndRead(ctx context.Context, dest *net.UDPAddr, msg *dhcpv6
 				}
 			}
 		}
-	})
+	}, backoff.WithMaxRetries(backoff.NewExponentialBackOff(), c.retry))
```

Comment:
You should probably also do a `backoff.WithContext(` here as well.

Reply:
Done.